### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by bio-boris** [kbase/.github#30]
# Go to the repo and press the actions button

<img width="691" alt="image" src="https://user-images.githubusercontent.com/1258634/193486895-98ae35a0-4600-41aa-bcab-c7aeab29fab2.png">

# Press "New Workflow"
<img width="691" alt="image" src="https://user-images.githubusercontent.com/1258634/193486976-3772de73-0475-448b-837e-68cf2c7bcce1.png">

# Press configure on each workflow you want to install

<img width="1148" alt="image" src="https://user-images.githubusercontent.com/1258634/193487011-75d47c00-ded3-464e-a8b4-a6fd6f1ec42f.png">

**Comment by MrCreosote** [comment]
FWIW I see a different UI in the actions tab for a brand new repo:
![image](https://user-images.githubusercontent.com/1686141/194191911-6bf6b507-c04b-4c91-be01-6adf7321476c.png)

**Comment by bio-boris** [comment]
ah that means the repo was created without using the kbase template, which should be used to create repos.

**Comment by bio-boris** [comment]
but yes could add that screenshot too

**Comment by MrCreosote** [comment]
I'm confused then - if you create with the KB template you don't need to go through the first part of this process because the actions will already be there, right? It's only if you don't use a template when you need to to that

**Comment by bio-boris** [comment]
That's right. If you don't use a template then you should probably delete your repo and use that repo as a template, or go through the above steps. Adding the screenshot

**Comment by bio-boris** [comment]
Admin also need to make it public as well
